diff --git a/howdy/src/meson.build b/howdy/src/meson.build
index 2cff83a..35d86d0 100644
--- a/howdy/src/meson.build
+++ b/howdy/src/meson.build
@@ -105,7 +105,7 @@ if not fs.exists(config_path)
     install_data('config.ini', install_dir: confdir, install_mode: 'rwxr--r--', install_tag: 'config')
 endif
 
-install_data('dlib-data/install.sh', 'dlib-data/Readme.md', install_dir: dlibdatadir, install_mode: 'rwxr--r--')
+#install_data('dlib-data/install.sh', 'dlib-data/Readme.md', install_dir: dlibdatadir, install_mode: 'rwxr--r--')
 
 install_man('../howdy.1')
 
